src/django/api/migrations/0177_add_new_certifications.py

```
@@ -0,0 +1,135 @@
+# Generated by Django 3.2.17 on 2025-09-05 17:38
+
+import django.contrib.postgres.fields
+import django.contrib.postgres.indexes
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0176_introduce_enable_dromo_uploading_switch'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)